default.json

```diff
@@ -1,46 +1,67 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "config:base",
+    "config:recommended",
     "github>Linkurious/renovate-config:meta",
     "workarounds:nodeDockerVersioning",
-    "regexManagers:dockerfileVersions"
+    "customManagers:dockerfileVersions"
+  ],
+  "baseBranchPatterns": [
+    "develop-maintenance",
+    "develop-stable",
+    "develop"
+  ],
+  "labels": [
+    "dependencies"
   ],
-  "baseBranches": ["develop-maintenance", "develop-stable", "develop"],
-  "labels": ["dependencies"],
   "branchConcurrentLimit": 10,
   "prConcurrentLimit": 2,
   "prHourlyLimit": 10,
   "respectLatest": true,
   "ignoreUnstable": true,
   "schedule": "every weekend",
-  "docker": {
-    "addLabels": ["docker"]
-  },
   "packageRules": [
     {
-      "matchDatasources": ["docker"],
-      "addLabels": ["docker"]
+      "matchDatasources": ["docker", "node-version"],
+      "matchPackagePatterns": ["node"],
+      "groupName": "node and docker node updates",
+      "groupSlug": "node-docker-updates"
```

Comment:
For the record, this was needed to fix node pr's to be grouped, due to precedence of rules.